### ISSUE TITLE
Ideas for the website.

### ISSUE THREAD

**Issue by dynamitechetan** [TheAlgorithms/website-old#4]
The website is pretty basic right now. We need some ideas that what we need on the website.

**Comment by nelsonmfinda** [comment]
Hello, I would like to contribute an idea to the website. I will analyze the site and then I will comment on my suggestions and make PR with my contribution.

**Comment by pranikamassey** [comment]
Hello, I would like to work on this website and work on the website interface. May I take this issue.

**Comment by ghost** [comment]
I would like to work on this issue.

**Comment by tasgmaia** [comment]
You guys could use this guy. https://react-styleguidist.js.org/
I know the purpose is kind of different but you can show each algorithm on the page.

**Comment by cdbm** [comment]
the link to github could be more easier to find, i could work on it

**Comment by vinay72** [comment]
There should be a video on the Official website of Hacktoberfest defining how to participate in Hactoberfest 2018 and win a free T-shirt by making 5 pull requests.

**Comment by dhanu10896** [comment]
we can use UI framework like bootstrap for starting.

**Comment by christianbender** [comment]
@dynamitechetan @AnshulMalik  Why we don't use a CSS-framework like [Bootstrap](https://getbootstrap.com/) or [W3.CSS](https://www.w3schools.com/w3css/) ?

**Comment by AnshulMalik** [comment]
That would be overkill @christianbender, considering our current requirement.

**Comment by christianbender** [comment]
@AnshulMalik OK I understand. How about a visitor counters? We don't need made it self. 
 For example:  
https://www.freevisitorcounters.com/
http://www.cutercounter.com/

**Comment by AnshulMalik** [comment]
Nope, I don't see any plus of putting a visitor counter. You think it may help in something?

**Comment by christianbender** [comment]
@AnshulMalik  We would have an overview of the visitors.

**Comment by AnshulMalik** [comment]
Yeah, but how is it adding value to our objective of helping people to educate about algorithms?

**Comment by carolinemusyoka** [comment]
Change in the website`s UI would be helpful, ie little bit more interactive using motion design and color change.

**Comment by mayur-samrutwar** [comment]
> Yeah, but how is it adding value to our objective of helping people to educate about algorithms?

I think adding a rating bar per algorithm will help a lot. Any newcomer will get to know which Algorithm is more hard or important.

**Comment by rishikeshjoshi123** [comment]
Sir I am new to git hub . I know C++ and currently I am learning OpenGL .Can you please guide me as to how to find some real stuff on github to contribute and learn . I am currently in my second year of college. I want to contribute to open source projects but since my skill level is very beginner I am not sure how to find projects .
